From 39e09e379d1511bdf4d9484f1ae0efdd516894fa Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Mon, 3 May 2021 15:35:38 +0000
Subject: [PATCH] Exclude ReadyToRun.Experimental NuGet package

Not required by SDK and depends on non-SB components (e.g. r2rdump)
---
 src/coreclr/.nuget/coreclr-packages.proj | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/coreclr/.nuget/coreclr-packages.proj b/src/coreclr/.nuget/coreclr-packages.proj
index 2f7d06a..b8aec3a 100644
--- a/src/coreclr/.nuget/coreclr-packages.proj
+++ b/src/coreclr/.nuget/coreclr-packages.proj
@@ -13,6 +13,9 @@
     <ProjectReference Include="Microsoft.NETCore.TestHost\Microsoft.NETCore.TestHost.proj" />
     <ProjectReference Include="Microsoft.NETCore.ILAsm\Microsoft.NETCore.ILAsm.proj" />
     <ProjectReference Include="Microsoft.NETCore.ILDAsm\Microsoft.NETCore.ILDAsm.proj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
     <ProjectReference Include="ILCompiler.Reflection.ReadyToRun.Experimental\ILCompiler.Reflection.ReadyToRun.Experimental.pkgproj" />
   </ItemGroup>
 
-- 
1.8.3.1

